handbook/engineering/code-intelligence/goals.md

```diff
@@ -0,0 +1,45 @@
+# Cloud goals and priorities
+
+## Goals
+
+Progress on adoption and usage is tracked in our [Looker dashboard](https://sourcegraph.looker.com/dashboards/131).
+
+### 95% of all code intel results are precise for any fully supported language
```

Comment:
The problem with this goal is that we can achieve it without actually working on the right things. Even if we have 100% for all of these, we will have failed if the denominator is small. We need actual adoption and usage of LSIF. If we build the perfect precise code intelligence but barely anyone uses it, we have failed, and we need a goal that reflects that.

A WAU metric would make sense to me here (and the milestones don't need to change if we think that is exactly what we need to do to drive WAU).

Reply:
WAU is nice because it encourages us to build things that are valuable to a broad audience (we will build LSIF Java before we build lsif lisp).